mpmath/ctx_base.py

```diff
@@ -357,6 +357,7 @@ def sum_accurately(ctx, terms, check_step=1):
             while 1:
                 ctx.prec = prec + extraprec + 5
                 max_mag = ctx.ninf
+                sum_mag = ctx.zero
```

Comment:
This is unrelated change, it lacks a test.  Lets fix https://github.com/mpmath/mpmath/issues/604 in https://github.com/mpmath/mpmath/pull/664.

Reply:
OK, what do you want me to do with this PR **exactly**?

Reply:
Lets just revert this:)

Reply:
Done. Now what?